pyproject.toml

```diff
@@ -28,6 +28,7 @@ classifiers = [
     "Programming Language :: Python :: 3.6",
     "Programming Language :: Python :: 3.7",
     "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
```

Comment:
I'm not going to add the 3.10 trove classifier until https://github.com/samuelcolvin/pydantic/pull/2885 is released

/cc @PrettyWood